docs/community/hash_collision_postmortem.md

```diff
@@ -0,0 +1,35 @@
+## POSTMORTEM June 9th, 2021
+
+### Quick summary
+
+On December 2, 2020, we noticed the determined way of creating [transaction hash collisions on Testnet](https://github.com/near/nearcore/security/advisories/GHSA-2v6r-g342-282f), then also on Mainnet.
+
+It still affects the data representation in some services such as [Near Explorer](https://explorer.near.org).
+
+[The fix](https://github.com/near/nearcore/pull/4064) was merged on 11 of March 2021 and released in [1.19.0](https://github.com/near/nearcore/releases/tag/1.19.0), so new examples of reproducible hash collisions will not appear again.
+
+### In details
+
+There was a possibility to apply the same transaction two or more times.
+Such transactions are logically different objects, but they have the same hash.
```

Comment:
Not sure what you mean by logically different objects -- they are the *same* transaction

Reply:
In a normal situation, the transaction could be applied only once. We have the situation when it is applied twice or more times. I needed to speak about different appearances. Sorry if that confuses you. 
Is it clear now what I wanted to say?
Could you please correct my wording?

Reply:
I think stating that a transaction should only be applied once is probably better

Reply:
This wording does not explain our case. It should be applied only once, but it's not true in the real world.

Reply:
Well what I meant is that it should be applied only once, but due to the issue we had, some transaction was applied more than once.

Reply:
@bowenwang1996 It seems we lost the shared context. This page is a postmortem about the transactions being included in two different blocks/chunks, and thus applied twice, and thus having two execution outcomes, got converted to two different receipts, and the following execution happened as if those transactions are different, but on Indexer for Explorer side, we cannot really disambiguate this scenario unless we introduce unnecessary redundancy which will confuse people even more than that.